client/app/styles/main.styl

```diff
@@ -732,20 +732,20 @@ body
         box-sizing: border-box
         width: 180px
         height: 180px
-        padding: 20px 20px 10px 20px
+        padding: 25px 20px 10px 20px
         margin-right: 10px
 
         img.svg
-            border-radius: 200px
-            padding: 10px
-            width:140px
-            height:140px
+            border-radius 200px
```

Comment:
to ensure that the border-radius is used to build a round shape, prefer to use `border-radius: 50%`. Angle will be rounded properly :wink:.